docs/watermark.ipynb

```diff
@@ -471,9 +483,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "scipy   1.3.1\n",
-      "numpy   1.17.2\n",
-      "sklearn 0.21.3\n",
+      "scipy: 1.5.0\n",
```

Comment:
The changes look fine overall. However, this one is a bit concerning. Any idea why numpy and sklearn are not shown anymore?

Reply:
Probably hard to see above, so here is the issue visually:

![Screen Shot 2020-10-29 at 10 46 05 AM](https://user-images.githubusercontent.com/5618407/97597581-fa1e0f80-19d3-11eb-98b1-e1fa9496bfeb.png)

Reply:
Hi there,
I checked the source code, it wasn't my fault. This happens because someone changed the source code to `hasattr` method here:
https://github.com/rasbt/watermark/blob/d0862950eb081809c69ada6095370ade4b2bdf53/watermark/watermark.py#L237

and this prevents the following line from happening while in the past it worked because at some point the `AttributeError` would raise:
https://github.com/rasbt/watermark/blob/d0862950eb081809c69ada6095370ade4b2bdf53/watermark/watermark.py#L240

You can see the whole process here:

https://github.com/rasbt/watermark/blob/d0862950eb081809c69ada6095370ade4b2bdf53/watermark/watermark.py#L236-L246

Do you want me to fix it?

Reply:
Yeah, you're totaly right about the code style, it's just a matter of tasts. One thing more, @rasbt, the default maximum line length should be 79.

Reply:
Good catch regarding the previous change. I wasn't careful when merging that PR. If you could fix it that would be nice (and also changing the 80 -> 79 in the)

Reply:
I fixed:
* The line length 
* And also the method that returns a list of all imported modules.

While I was fixing those problems, I noticed that I misundertood the time and dating options. I made a mistake, currently, if user enters `-d` or `-t` one of them will be used. I'll fix that also soon.

Reply:
> I fixed:
> - The line length
> - And also the method that returns a list of all imported modules.

Awesome. This works now. I updated the readme and jupyter notebook.

>  if user enters -d or -t one of them will be used. I'll fix that also soon.

no worries. but it would be good to fix this before we merge.

Reply:
Hello there,

I think I fixed the problem, I testd it multiple times with these options:
-   -d
-   -n
-   -t
-   -i
-   -z
-   -u
- -c

While, there is this possibility that ff a user enters `-u` without any other options, in this case, it will show something like this:
```bash
Last updated: 
```
Other than this, rare case, everything works pretty good!

Reply:
Awesome, thanks a lot!

Reply:
My pleasure!